staging/src/k8s.io/component-base/logs/options.go

```diff
@@ -50,7 +53,8 @@ func (o *Options) Validate() []error {
 
 // AddFlags add logging-format flag
 func (o *Options) AddFlags(fs *pflag.FlagSet) {
-	fs.StringVar(&o.LogFormat, logFormatFlagName, defaultLogFormat, "Set log format")
+	formats := strings.Join(logRegistry.List(), `", "`)
+	fs.StringVar(&o.LogFormat, logFormatFlagName, defaultLogFormat, fmt.Sprintf(`Set format of logs written by component. Options: "%s". Non default options don't support Global log flags.`, formats))
```

Comment:
> Non default options don't support Global log flags.

I don't think users can easily figure out the meaning.  Here is an example, I think it's better.

```
--audit-log-format string     Default: "json"
Format of saved audits. "legacy" indicates 1-line text format for each event. "json" indicates structured json format. Known formats are legacy,json.
```

Reply:
Listed all unsupported flags